### PR TITLE
argos-translate: add page

### PR DESCRIPTION
**Pull request by ikks**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/argos-translate.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# argos-translate
+
+> An Open-source offline translation library written in Python. Also CLI tool.
```
Comment (kbdharun) [inline pages/common/argos-translate.md:3]:
```suggestion
> An Open-source offline translation library and CLI tool written in Python.
```
How about this?

File: pages/common/argos-translate.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+> More information: <https://www.argosopentech.com/>.
+
+- Install translation pair from Spanish to English:
```
Comment (kbdharun) [inline pages/common/argos-translate.md:6]:
```suggestion
- Install translation pairs for Spanish to English translation:
```

File: pages/common/argos-translate.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+`argospm install translate-es_en`
+
+- Translate some text from Spanish (es) to English (en). Mind to use only two letter codes:
```
Comment (kbdharun) [inline pages/common/argos-translate.md:10]:
```suggestion
- Translate some text from Spanish (es) to English (en) (Note: only two letter language codes are supported):
```

File: pages/common/argos-translate.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+`argos-translate --from-lang es --to-lang en {{un texto corto}}`
+
+- Translate a text file from English to Hindi:
+
+`cat {{text_file.txt}} | argos-translate --from-lang en --to-lang hi`
```
Comment (kbdharun) [inline pages/common/argos-translate.md:16]:
```suggestion
`cat {{path/to/file.txt}} | argos-translate --from-lang en --to-lang hi`
```

File: pages/common/argos-translate.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+- List of installed translation pairs:
```
Comment (kbdharun) [inline pages/common/argos-translate.md:18]:
```suggestion
- List all installed translation pairs:
```

File: pages/common/argos-translate.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+`argospm list`
+
+- Show translation pairs from English available to be installed:
```
Comment (kbdharun) [inline pages/common/argos-translate.md:22]:
```suggestion
- Show translation pairs from English that are available to be installed:
```

File: pages/common/argos-translate.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+`argospm search --from-lang en`
+
+- Update installed language package pairs:
+
+`argospm update`
+
+- Translate from `ar` to `ru`. When the translation pairs `translate-es_en` and `translate-en_ru` are installed:
```
Comment (kbdharun) [inline pages/common/argos-translate.md:30]:
```suggestion
- Translate from `ar` to `ru` (Note: this requires the translation pairs `translate-es_en` and `translate-en_ru` to be installed):
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/common/argos-translate.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+- Show translation pairs from English that are available to be installed:
+
+`argospm search --from-lang en`
+
+- Update installed language package pairs:
+
+`argospm update`
+
+- Translate from `ar` to `ru` (Note: this requires the translation pairs `translate-es_en` and `translate-en_ru` to be installed):
```
Comment (spageektti) [inline pages/common/argos-translate.md:30]:
```suggestion
- Translate from `ar` to `ru` (Note: this requires the translation pairs `translate-ar_en` and `translate-en_ru` to be installed):
```

File: pages/common/argos-translate.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
+# argos-translate
+
+> An Open-source offline translation library and CLI tool written in Python.
+> More information: <https://www.argosopentech.com/>.
+
+- Install translation pairs for Spanish to English translation:
+
+`argospm install translate-es_en`
+
+- Translate some text from Spanish (es) to English (en) (Note: only two letter language codes are supported):
```
Comment (spageektti) [inline pages/common/argos-translate.md:10]:
```suggestion
- Translate some text from Spanish (`es`) to English (`en`) (Note: only two letter language codes are supported):
```

File: pages/common/argos-translate.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
+# argos-translate
+
+> An Open-source offline translation library and CLI tool written in Python.
```
Comment (acuteenvy) [inline pages/common/argos-translate.md:3]:
```suggestion
> An open-source offline translation library and CLI tool written in Python.
```

**Review by acuteenvy — APPROVED** [review]

**Review by spageektti — APPROVED** [review]

**Pull request merged**